sbhs/views.py

```diff
@@ -372,10 +393,14 @@ def slot_new(request):
 
     else:
         form = SlotCreationForm()
+        avail_slot = Slot.objects.check_booked_slots(now,all_board_users_id)
         context['history']=slot_history
         context['form']=form
         context['now'] = now
+        context['duration'] = settings.SLOT_DURATION
```

Comment:
It will be better if we have sbhs settings in a separate file and not in django settings